src/views/Orders.vue

```diff
@@ -176,22 +179,34 @@ export default defineComponent({
         this.getPackedOrders().then(() => { event.target.complete() });
       }
     },
-    async viewOrder (order: any) {
+    async viewOrder (order: any, shipmentMethod: any) {
       // TODO: find a better approach to handle the case that when in open segment we can click on
       // order card to route on the order details page but not in the packed segment
-      await this.store.dispatch('order/updateCurrent', { order }).then(() => {
-        this.$router.push({ path: `/orderdetail/${order.orderId}` })
+      await this.store.dispatch('order/updateCurrent', { order, shipmentMethod }).then(() => {
+        this.$router.push({ path: `/orderdetail/${order.orderId}/${shipmentMethod}` })
       })
     },
     async getPickupOrders (vSize?: any, vIndex?: any) {
       const viewSize = vSize ? vSize : process.env.VUE_APP_VIEW_SIZE;
       const viewIndex = vIndex ? vIndex : 0;
+
       const payload = {
-        sortBy: 'orderDate',
-        sortOrder: 'Desc',
+        "inputFields":{
+          "orderTypeId": "SALES_ORDER",
+          "orderItemStatusId": "ITEM_APPROVED",
```

Comment:
We  will need ITEM_APPROVED and ITEM_CREATED items. Please update condition

Reply:
Updated Sir.